### DISCUSSION TITLE
리트코드 스터디에 오신 것을 환영합니다

### DISCUSSION THREAD

**Discussion by DaleSeo** [DaleStudy/leetcode-study#9] [공지사항]
안녕하세요 여러분, 👋 저는 달레입니다.

스터디에 참여자 분들이 대부분 개발자이실 것 같아서 개발자답게 GitHub를 최대한 활용해서 스터디를 진행하려고 합니다. 스터디 활동을 통해서 예비 개발자분들께서 유용한 소프트웨어 개발 플랫폼인 GitHub와 친해지셨으면 좋겠다는 마음도 있고요. 😄

GitHub에 올라오는 공지사항은 디스코드 채널에도 올라오도록 연동을 해놓았습니다. 아무래도 디스코드에 공지를 올리면 대화를 하면서 계속 위로 밀려나기 때문에 나중에 찾기가 매우 힘들어지더라고요. 디스코드 채널은 스터디 멤버끼리 편하게 대화를 하는 용도로 사용하고, 스터디 관련 중요한 공지는 GitHub 통해서 올리도록 하겠습니다.

그럼 오늘도 즐거운 코딩하세요! 👨‍💻👩‍💻

**Comment by WhiteHyun** [comment]
안녕하세요 달레님! 좋은 기회로 스터디를 만들어주신 점 정말 감사합니다.
최선을 다해 참여하도록 할게요! 🎉🎉
